src/Webapi/Webapi__Fetch.res

```diff
@@ -366,10 +294,10 @@ module RequestInit = {
       ~body?,
       ~referrer?,
       ~referrerPolicy=encodeReferrerPolicy(referrerPolicy),
-      ~mode=?map(encodeRequestMode, mode),
-      ~credentials=?map(encodeRequestCredentials, credentials),
-      ~cache=?map(encodeRequestCache, cache),
-      ~redirect=?map(encodeRequestRedirect, redirect),
```

Comment:
I had no idea this `=?map` syntax was valid 🤨 this was just how it converted from the original OCaml. Looks like it's a convenient way to pass an optional function return value as an optional method argument 🤔 

Learn something new every day, I guess!